src/commands/solana.ts

```diff
@@ -0,0 +1,26 @@
+import { Command } from "@oclif/core";
+
+import inquirer from "inquirer";
+
+import sendTokens from "../lib/sendTokens";
+import getAddressFromConfigFile from "../utils/getAddressFromConfigFile";
+
+import solQuestions from "../data/solQuestions";
+import createOrUpdateConfigFile from "../utils/createOrUpdateConfigFile";
+
+export default class Solana extends Command {
+  static description = "🦄 Get Solana by using the solana command";
```

Comment:
It would be better to have this has the description "Get devnet SOL by using the solana command" as the current one is kinda misleading.